_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -504,6 +504,63 @@ public static function register_endpoints() {
 				'permission_callback' => __CLASS__ . '::view_admin_page_permission_check',
 			)
 		);
+
+		/*
+		 * Get and update settings from the Jetpack wizard.
+		 */
+		register_rest_route(
+			'jetpack/v4',
+			'/setup/questionnaire',
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => __CLASS__ . '::get_setup_questionnaire',
+					'permission_callback' => __CLASS__ . '::update_settings_permission_check',
+				),
+				array(
+					'methods'             => WP_REST_Server::EDITABLE,
+					'callback'            => __CLASS__ . '::update_setup_questionnaire',
+					'permission_callback' => __CLASS__ . '::update_settings_permission_check',
+					'args'                => array(
+						'option_values' => array(
+							'required' => true,
+							'type'     => 'object',
+						),
+					),
+				),
+			)
+		);
+	}
+
+	/**
+	 * Update the settings selected on the wizard questionnaire
+	 *
+	 * @param WP_REST_Request $request The request.
+	 *
+	 * @return bool true.
+	 */
+	public static function update_setup_questionnaire( $request ) {
+		// TODO: add validation.
+
+		$option_values = $request['option_values'];
+		Jetpack_Options::update_option( 'setup_questionnaire', $option_values );
+		return true;
+	}
+
+	/**
+	 * Get the settings for the wizard questionnaire
+	 *
+	 * @param WP_REST_Request $request The request.
+	 *
+	 * @return array Questionnaire settings.
+	 */
+	public static function get_setup_questionnaire( $request ) {
+		$options_values = Jetpack_Options::get_option( 'setup_questionnaire' );
```

Comment:
Instead of the `if`, you could just pass in `array()` as the second argument for `Jetpack_Options::get_option()`, and that will return `array()` as the default.

That would make the code look like:

```
return Jetpack_Options::get_option( 'setup_questionnaire', array() );
```